php/php.blade/src/org/netbeans/modules/php/blade/editor/EditorUtils.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.netbeans.modules.php.blade.editor;
+
+import javax.swing.text.Document;
+import org.netbeans.api.annotations.common.CheckForNull;
+import org.netbeans.api.project.Project;
+import org.netbeans.modules.editor.NbEditorUtilities;
+import org.netbeans.modules.php.blade.project.ProjectUtils;
+import org.openide.filesystems.FileObject;
+import org.openide.loaders.DataObject;
+
+/**
+ *
+ * @author bogdan
+ */
+public final class EditorUtils {
+
+    private EditorUtils() {
+    }
+
+    @CheckForNull
+    public static FileObject getFileObjectFromDoc(Document doc) {
+        DataObject dObject = NbEditorUtilities.getDataObject(doc);
+        if (dObject != null) {
+            return dObject.getPrimaryFile().getParent();
```

Comment:
Is the method name proper?

Reply:
@haidubogdan Again, Is the method name proper? This method returns a parent directory of a document, right?
`getFileObject` is expected that a file object of the document is returned, I think.